marshal.go

```diff
@@ -2280,7 +2282,7 @@ func marshalUDT(info TypeInfo, value interface{}) ([]byte, error) {
 	}
 
 	if k.Kind() != reflect.Struct || !k.IsValid() {
-		return nil, marshalErrorf("cannot marshal %T into %s", value, info)
+		return nil, marshalErrorf("cannot marshal %T into %s. Accepted types: Marshaler, UDTMarshaler, map[string]interface{}, struct, UnsetValue.", value, info)
```

Comment:
It looks like `struct` is also accepted here

Reply:
Explain please, because `struct` is already here.

Reply:
Oh I'm sorry, I was probably looking at the wrong place